Thread Control Issues

dbknet opened the pull request:
Hi,
Recently I had the opportunity to use your library. Good job!
While using ThreadControl, I noticed several issues, which is why I made a pull request with changes.
I hope it will be useful.

ker0x commented on src/Model/Callback/PassThreadControl.php:
```diff
@@ -19,12 +19,12 @@ class PassThreadControl
+     * @param string|null $metadata
      */
-    public function __construct(string $newOwnerAppId, string $metadata)
+    public function __construct($newOwnerAppId, ?string $metadata)
```
What's the type of $newOwnerAppId ? If it's an int that is always return then in this case, can you add int for type hinting instead of nothing, removing the casting into a string and change the return type of the getter please ?

ker0x commented on src/Model/Callback/RequestThreadControl.php:
```diff
@@ -19,12 +19,12 @@ class RequestThreadControl
+     * @param string|null $metadata
      */
-    public function __construct(string $requestedOwnerAppId, string $metadata)
+    public function __construct($requestedOwnerAppId, ?string $metadata)
```
Same here

dbknet commented:
@ker0x, thank you for your tips. I've corrected the returned types. I also made changes in the tests. In addition, I also added the TakeThreadControl file to make the entire ThreadControl Flow uniform.
